plugins/repository.py

```diff
@@ -74,17 +74,18 @@ def load_project(project, config):
 def get_linux_distro_version():
     # Handle both: distro module and old platform
     try:
-        return distro.linux_distribution()[2]
+        return distro.codename()
```

Comment:
I wonder if it is worth converting this to downcase either here or where it's consumed by the tooling which is generating the deb repository configuration so that we're robust to future changes to the codename format?